src/components/Actions/Actions.vue

```diff
@@ -373,6 +419,26 @@ import DotsHorizontal from 'vue-material-design-icons/DotsHorizontal.vue'
 
 const focusableSelector = '.focusable'
 
+const getActions = (ctx) => {
+	/**
+	 * Filter the Actions, so that we only get allowed components.
+	 * This also ensure that we don't get 'text' elements, which would
+	 * become problematic later on.
+	 */
+	if (ctx.$slots.default !== undefined) {
+		return ctx.$slots.default.filter(action => action?.componentOptions?.tag)
+	}
+
+	/**
+	 * Expose some internal methods to parent template
+	 */
+	if (ctx.$scopedSlots.default) {
+		return ctx.$scopedSlots.default({ clearFocusTrap: ctx.clearFocusTrap })
```

Comment:
Don't we need to filter the Actions form the scoped slot as well?

Reply:
Not for my point of view.
I do not know why we are doing this filter